_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+
+
+## Mute button
+
+This release doesn't stop at folders though. Thanks to [@karkaminski](https://github.com/karkaminski), NewPipe now has mute button. [@B0pol](https://github.com/B0pol) formed it further into the shape you can experience it in now. _"But where can I find it"_ you ask. Fear not, for I have prepared a gallery of screenshots. To access the pop-up and background player menus shown, you need to press on the notification from NewPipe.
+
+<img class="no-flow vertical" src="/img/screenshots/mute_main.jpg"/>
+<img class="no-flow img-responsive" src="/img/screenshots/mute_back.jpg"/>
+
+
+## Smaller Improvements
+
+[@comradekingu](https://github.com/comradekingu) took care of the English base strings in this release and made sure they are consistent. Further improving the user experience was also [@XiangRongLins](https://github.com/XiangRongLin) goal, he made sure that the seek duration jumped visibly to an even one when the fast inexact search option gets activated. Previously the user wouldn't see this directly, but experience when seeking for- or backwards. Thanks to [@opusforlife2](https://github.com/opusforlife2), two settings moved from `Appearance` to `Content`, where they belonged all along: `Show 'Next'` and `Content of main page`.
+
+[@wb9688](https://github.com/wb9688) and [@B0pol](https://github.com/B0pol) (in his second contribution) added support for more subdomains. Now `music.youtube.com` and `media.ccc.de` are natively supported by NewPipe, thanks you two. In the last point to this section, [@jonas-skywalker](https://github.com/jonas-skywalker) (may the force be with you) improved the search section. Now the magnifying glass vanishes once you start typing.
+
+## Fixes a.k.a. NewPipe crashes less \o/
+
+Android 9 changed how databases behave, effectively destroying backups and exporting remote playlists - not anymore, thanks to [@yausername](https://github.com/yausername). [@XiangRongLin](https://github.com/XiangRongLin) improved sharing a downloaded video to other apps, which crashed the app before this release. The app also crashed when you tried to download a really large video, which [@XiangRongLin](https://github.com/XiangRongLin) fixed in his third contribution. The seek duration setting was not properly shown in any other language then English, but [@B0pol](https://github.com/B0pol) let them magically appear again. And finally, [@kapodamy](https://github.com/kapodamy) took care of a nasty bugs, which made some WebM videos non seekable. Now you can seek all you want.
```

Comment:
> The app also crashed when you tried to download a really large video, which [@XiangRongLin](https://github.com/XiangRongLin) fixed in his third contribution

I don't remember doing this. I think you mixed it up with someone else

Reply:
Correct. This was kapodamy

Reply:
grrr. Reality should be what I want.